nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -48,6 +48,7 @@
 import org.projectnessie.api.TreeApi;
 import org.projectnessie.client.NessieClient;
 import org.projectnessie.client.NessieConfigConstants;
+import org.projectnessie.client.http.HttpClientException;
```

Comment:
The change to this file should be a different PR. Its cosmetic and unrelated to the bug we are fixing. It is a good cosmetic change and shoudl be done but imho not here,